lib/SGN/Controller/Search/Feature.pm

```diff
@@ -37,11 +46,13 @@ sub oldsearch : Path('/feature/search') Args(0) {
 
 sub search :Path('/search/features') Args(0) {
```

Comment:
There's two /search/features paths? (the others in SGN::Controller::Feature above). maybe one should be deprecated